packages/fuselage/src/components/Card/CardControls.tsx

```diff
@@ -1,5 +1,16 @@
-const CardControls = ({ ...props }) => (
-  <div className='rcx-card__controls' {...props} />
-);
+import { useBreakpoints } from '@rocket.chat/fuselage-hooks';
+import Box from '../Box/Box';
+
+const CardControls = ({...props }) => {
+  const breakpoints = useBreakpoints();
+  const isMobile = !breakpoints.includes('sm');
+  return (
+    <Box
+      rcx-card__controls
+      rcx-card__controls--wrap={isMobile}
+      {...props}
+    />
+  )
+}
```

Comment:
```suggestion
import { useBreakpoints } from '@rocket.chat/fuselage-hooks';

import Box from '../Box/Box';

const CardControls = ({ ...props }) => {
  const breakpoints = useBreakpoints();
  const isMobile = !breakpoints.includes('sm');
  return (
    <Box rcx-card__controls rcx-card__controls--wrap={isMobile} {...props} />
  );
};
```

Same thing as the last PR, there are prettier issues with this snippet.